src/v2/Apps/Settings/Routes/EditProfile/UserInformation.tsx

```diff
@@ -93,43 +99,48 @@ export const UserInformation: React.FC<UserInformationProps> = ({
           values,
         }) => (
           <Form onSubmit={handleSubmit}>
-            <Text>Full name</Text>
-            <QuickInput
+            <Input
+              mb={2}
+              title="Full name"
+              name="name"
               error={errors.name as any}
               placeholder="Enter your full name"
-              name="name"
               type="text"
               value={values.name}
               onChange={handleChange}
               onBlur={handleBlur}
+              width="50%"
             />
-            <Text>Email</Text>
-            <QuickInput
-              block
+            <Input
+              mb={2}
+              title="Email"
+              name="email"
               error={errors.email as any}
               placeholder="Enter your email address"
-              name="email"
               type="email"
               value={values.email}
               onChange={handleChange}
               onBlur={handleBlur}
+              width="50%"
```

Comment:
We should use the grid system on the outside of the form, rather than setting individual widths. The edit profile route should have an example.

Reply:
I moved this `UserInformation` Component into the existing grid component housing the other components on `SettingsEditProfileRoute` to take advantage of implemented styling.